### PR TITLE
prepare release 0.5.0

### PR DESCRIPTION
**Pull request by folkertdev**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,20 +1,21 @@
 [package]
 edition = "2021"
 name = "bzip2"
-version = "0.4.4"
+version = "0.5.0"
 authors = ["Alex Crichton <alex@alexcrichton.com>"]
```
Comment (bjorn3) [inline Cargo.toml:5]: A minor thing, but using the authors field is no longer encouraged and both crates.io and docs.rs no longer show it anywhere: https://rust-lang.github.io/rfcs/3052-optional-authors-field.html Maybe ask Alex if he wants to keep this field around or not?

Comment (folkertdev) [reply Cargo.toml:5]: I think we can just get rid of it then; github keeps track of the score

File: Cargo.toml [hunk 2]
```diff
@@ -1,20 +1,20 @@
 [package]
 edition = "2021"
 name = "bzip2"
-version = "0.4.4"
-authors = ["Alex Crichton <alex@alexcrichton.com>"]
+version = "0.5.0"
 license = "MIT/Apache-2.0"
```
Comment (squell) [inline Cargo.toml:5]:
```suggestion
license = "MIT OR Apache-2.0"
```
This is the correct syntax:
> If 2 or more licenses apply to a file, use an SPDX license expression. It is a composite expression constructed using parentheses, AND / OR operators, and the WITH operator for license exceptions (see below).

**Review by bjorn3 — APPROVED** [review]
I think we're ready to cut a new release.

**Comment by alexcrichton** [comment]
👍

**Pull request merged**